app/views/devise/registrations/new.html.erb

```diff
@@ -1,46 +1,53 @@
-<h1><%= t('.sign_up') %></h1>
-
-<%= form_for(resource, as: resource_name, url: registration_path(resource_name)) do |f| %>
-  <%= bootstrap_devise_error_messages! %>
-
-  <div class="form-group">
-    <%= f.label :email %>
-    <%= f.email_field :email, autofocus: true, autocomplete: 'email', class: 'form-control' %>
-  </div>
-
-  <div class="form-group">
-    <div class="col-md-6 no-padding">
-      <%= f.label :first_name %>
-      <%= f.text_field :first_name,  class: 'form-control' %>
-    </div>
-    <div class="col-md-6 no-padding left-padding">
-      <%= f.label :last_name %>
-      <%= f.text_field :last_name,  class: 'form-control' %>
-    </div>
+<div class="card panel-default">
+  <div class="card-header">
+    <h4><%= t('.sign_up', :default =>"Sign up") %></h4>
```

Comment:
Modify the hash syntax.
:default =>"Sign up" needs to be changed as following:
default: "Sign up"